packages/xo-web/src/xo-app/about/index.js

```diff
@@ -30,46 +27,97 @@ const HEADER = (
   user: getUser,
 }))
 export default class About extends Component {
-  componentWillMount() {
+  async componentWillMount() {
     serverVersion.then(serverVersion => {
       this.setState({ serverVersion })
     })
+
+    if (process.env.XOA_PLAN > 4 && COMMIT_ID !== '') {
+      try {
+        const commit = await getMasterCommit()
+        const isOnLatest = commit.sha === COMMIT_ID
```

Comment:
What if the user created a branch from master to tweak XO somehow? Then the commit ID won't be equal to `commit.sha` but they might also not be behind master (and actually a few commits ahead). I'm not sure what we should do in that case, do you think we should still show that they're not up-to-date with master? And saying "0 commits behind is acceptable"?

Reply:
As long as they are not on the same commit as master, I think it is necessary to say that they are not up to date (even if they have 0 commits behind).
In that case, why not just show "Not up-to date with master, n commit behind, n commit ahead"?

Reply:
Sounds good!